tests/elliptic_curves/test_ec_arithmetic.py

```diff
@@ -182,99 +224,95 @@ class Secp256r1Extension:
         curve_a=[0xFFFFFFFF00000001000000000000000000000000FFFFFFFFFFFFFFFFFFFFFFFC, 0],
         fq2=Fq2ScriptModel(q=modulus, non_residue=NON_RESIDUE_R1.to_list()[0]),
     )
-    test_positions = True
+    positions_addition = [
```

Comment:
maybe it is worth mentioning at some point that we are checking all the combinations of operations